### PR TITLE
Fixes clothing fixing one more time

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
I have no clue how this happened but somehow the last 3 fixes between myself and shizcalev have not actually fixed repairing clothing. I swear I tested it before on the last PR, but maybe I just imagined that and now have egg on my face. Whatever, here's proof I tested it this time.

[](https://i.imgur.com/t3GmJT1.png)

You can now actually use cloth to repair shredded and damaged clothing. Also made it so your sensors didn't become permanently broken if they're fully shredded then repaired.

## Why It's Good For The Game
puts this demon bug to bed

## Changelog
:cl: Ryll/Shaps
fix: You can now mend damaged clothing with cloth. I swear, this time you really can.
/:cl:

### CODE REVIEW

File: code/modules/clothing/clothing.dm [hunk 1]
```diff
@@ -358,7 +358,8 @@
 			to_chat(M, "<span class='warning'>[src] start[p_s()] to fall apart!</span>")
 
 //This mostly exists so subtypes can call appriopriate update icon calls on the wearer.
-/obj/item/clothing/proc/update_clothes_damaged_state()
+/obj/item/clothing/proc/update_clothes_damaged_state(damaged_state = CLOTHING_DAMAGED)
+	damaged_clothes = damaged_state
 	return
```
Comment (ShizCalev) [inline code/modules/clothing/clothing.dm:363]: not needed

Comment (Ryll-Ryll) [reply code/modules/clothing/clothing.dm:363]: It is, this is the entire fix

Comment (ShizCalev) [reply code/modules/clothing/clothing.dm:363]: i mean the return at the bottom isn't needed. it's gonna return anyway.

Comment (Ryll-Ryll) [reply code/modules/clothing/clothing.dm:363]: Oh lol, right.....

**Comment by ATH1909** [comment]
it's the skeletons+milk interaction all over again

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**